.pylintrc

```diff
@@ -0,0 +1,627 @@
+[MAIN]
```

Comment:
i generated this file using pylint and then added in the custom settings we had setup in our toml files.